README.md

```diff
@@ -56,13 +59,15 @@ Ensure that you have the following installed
   * C++ compiler
   * [clang-format](https://clang.llvm.org/docs/ClangFormat.html)
   * [CMake v3.16+](https://cmake.org/)
-  * [Qt 5 or Qt 6 (experimental)](https://www.qt.io/)
-  * [Rust](https://www.rust-lang.org/)
-  * Linux 64-bit x86 - currently we only support Linux, but we plan on adding arm 64-bit, macOS, and Windows support in the future
+  * [Qt 5 and/or Qt 6](https://www.qt.io/)
+  * [Rust toolchain](https://www.rust-lang.org/)
+  * [mold](https://github.com/rui314/mold) or [lld](https://lld.llvm.org/) for Linux (lld is included in the XCode toolchain on macOS)
 
-### Compiling
-In a CXX-Qt project, the build system is based on CMake, which uses Cargo under the hood.
-Therefore, unlike a typical Rust project, CMake must be used to build CXX-Qt.
+This repository's build system uses CMake, which calls Cargo under the hood to build all the
```

Comment:
Are the individual examples still buildable with CMake by themselves ? and should we mention this ?

Eg can you enter the `qml_minimal` folder and do `mkdir build; cd build; cmake ..; make; ./qml_minimal` ?

Reply:
They should be, but I don't think we need to mention this. There's already a lot of text in this section.